dashboard/app/models/concerns/text_to_speech.rb

```diff
@@ -229,5 +229,12 @@ def tts_update
       end
       self.authored_hints = JSON.dump(hints)
     end
+
+    # if this level is contained in another level, updating it should also
+    # trigger updates in its parents, since their content is likely at least
+    # partially based on this
+    parent_levels.contained.each do |containing_level|
```

Comment:
Just for my education -- is the parent-child relationship many-to-many?  (I kind of expected this to be `parent_level` instead of `parent_levels`?)

Reply:
yep! We've got quite a few levels which have several parents; assessment LevelGroups in particular tend to reuse children quite a bit.